scripts/docker/Dockerfile.openmptarget

```diff
@@ -55,7 +55,6 @@ RUN LLVM_URL=https://github.com/llvm/llvm-project/archive &&\
       -DCMAKE_CXX_COMPILER=g++ \
       -DLLVM_ENABLE_PROJECTS="clang" \
       -DLLVM_ENABLE_RUNTIMES="openmp" \
-      -DCLANG_OPENMP_NVPTX_DEFAULT_ARCH=sm_70 \
       -DLIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES=70 \
```

Comment:
Don't you want to use
````
-DLIBOMPTARGET_DEVICE_ARCHITECTURES=sm_70
```
instead?

Reply:
No, this is sufficient `-DLIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES=70 ` . The default arch option is getting ignored actually.

Reply:
Would you post a link to a reference for the build options doc?

Reply:
This is the best documentation I found [wiki](https://hpc-wiki.info/hpc/Building_LLVM/Clang_with_OpenMP_Offloading_to_NVIDIA_GPUs) .
While it still says use the `DEFAULT_ARCH`, cmake actually ignores it. So I guess the documentation is not up-to date

Reply:
I was looking at https://openmp.llvm.org/SupportAndFAQ.html#q-how-to-build-an-openmp-nvidia-offload-capable-compiler.

Reply:
Wait the doc says to use `-DLIBOMPTARGET_DEVICE_ARCHITECTURES=sm_<xy>`

Reply:
We should be using `LIBOMPTARGET_DEVICE_ARCHITECTURES` (confirmed on the llvm/openmp discord channel)